folly/memory/UninitializedMemoryHacks.h

```diff
@@ -21,11 +21,30 @@
 #include <type_traits>
 #include <vector>
 
-// On MSVC an incorrect <version> header get's picked up
+// On MSVC an incorrect <version> header gets picked up
 #if !defined(_MSC_VER) && __has_include(<version>)
 #include <version>
 #endif
 
+#if defined(_MSC_VER) && !defined(_M_CEE_PURE) && !defined(_DISABLE_STRING_ANNOTATION)
```

Comment:
Is `_DISABLE_STRING_ANNOTATION` some well-known symbol? If not and this is supposed to be specific to folly, then it must be namespaced to folly.